gobblin-service/src/test/java/org/apache/gobblin/service/monitoring/GaaSObservabilityProducerTest.java

```diff
@@ -112,9 +118,9 @@ public void testCreateGaaSObservabilityEventWithFullMetadata() throws Exception
     Assert.assertEquals(event.getJobStatus(), JobStatus.SUCCEEDED);
     Assert.assertEquals(event.getExecutorUrl(), "hostName");
     Assert.assertEquals(event.getIssues().size(), 1);
-    Assert.assertEquals(event.getFlowEdgeId(), "flowEdge");
+    Assert.assertEquals(event.getFlowEdgeId(), "sourceNode_destinationNode_flowEdge");
```

Comment:
don't we want this to be merely `flowEdge` (since we've already separately tunneled `sourceNode` and `destinationNode`)?

(I realize you're merely checking what you inserted just above... more I want to ensure the real events emitted actually have solely the flow edge ID as a stand-alone string)

Reply:
Made the change via parsing with StringUtils